internal/olm/operator/registry/fbcindex/fbc_registry_pod.go

```diff
@@ -245,8 +252,17 @@ func (f *FBCRegistryPod) podForBundleRegistry(cs *v1alpha1.CatalogSource) (*core
 							SubPath:   cm.Name,
 						},
 					},
+					SecurityContext: &corev1.SecurityContext{
+						Privileged:               pointer.Bool(false),
+						ReadOnlyRootFilesystem:   pointer.Bool(false),
```

Comment:
Why are we adding these ones?

Reply:
This is for adding securityContext to the container as well, along with the pod and setting them to false.